ooniprobe/View/RunTestsViewController.mm

```diff
@@ -137,6 +145,12 @@ -(void)showToast{
     [self.view makeToast:NSLocalizedString(@"ooniprobe_configured", nil) duration:3.0 position:CSToastPositionBottom style:style];
 }
 
+-(void)showToast:(NSNotification *) notification{
```

Comment:
Since my understanding is that this is the toast showed at end of test, maybe you can call it `showToastTestFinished`?

Reply:
Oh, I see that name already exists. This confused me, probably: what is the purpose of this toast?

Reply:
mmm, `git grep shotToastTestFinished` does not return any row tho.

Reply:
Disregard, now I understand. The `showToast` is the generic show toast function (maybe rename it `showToastGeneric`?) and the one above is specific for configuration. The `@"showToastTestFinished"` is just the name of a specific notification that you're using inside the app (i.e. it is a string, not a function).

Reply:
^^^ @lorenzoPrimi is it correct what I am saying here?

Reply:
Yes